studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.utils.tsx

```diff
@@ -36,6 +36,38 @@ export const generateProductRoutes = (ref?: string, project?: ProjectBase): Rout
         ref &&
         (isProjectPaused ? homeUrl : isProjectBuilding ? buildingUrl : `/project/${ref}/editor`),
     },
+    {
+      key: 'sql',
+      label: 'SQL Editor',
+      icon: (
+        <SVG
+          src="/img/sql-editor.svg"
+          style={{ width: `${18}px`, height: `${18}px` }}
+          preProcessor={(code) => code.replace(/svg/, 'svg class="m-auto text-color-inherit"')}
+        />
+      ),
+      link:
+        ref &&
+        (isProjectPaused ? homeUrl : isProjectBuilding ? buildingUrl : `/project/${ref}/sql`),
+    },
+  ]
+}
+export const generateProductRoutes = (ref?: string, project?: ProjectBase): Route[] => {
+  const isProjectBuilding = project?.status !== PROJECT_STATUS.ACTIVE_HEALTHY
+  const isProjectPaused = project?.status === PROJECT_STATUS.INACTIVE
+
+  const homeUrl = `/project/${ref}`
+  const buildingUrl = `/project/${ref}/building`
+
+  return [
+    {
+      key: 'database',
+      label: 'Database',
+      icon: <IconDatabase size={18} strokeWidth={2} />,
+      link:
+        ref &&
+        (isProjectPaused ? homeUrl : isProjectBuilding ? buildingUrl : `/project/${ref}/editor`),
```

Comment:
wrong URL

Reply:
hummm isn't ref/editor the right url there?